unresolved-package-handler/base/deploymentconfig.yaml

```diff
@@ -0,0 +1,105 @@
+---
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  annotations:
+      thoth-station.ninja/template-version: 0.1.0
+  labels:
+    app: thoth
+    component: unresolved-package-handler-consumer
+  name: unresolved-package-handler-consumer
+spec:
+  replicas: 1
+  selector:
+    service: unresolved-package-handler-consumer
```

Comment:
selector needs to include https://github.com/thoth-station/thoth-application/blob/master/core/base/kustomization.yaml#L4-L6 see https://github.com/thoth-station/thoth-application/issues/13